t/unit-tests/u-oidmap.c

```diff
@@ -0,0 +1,152 @@
+#include "unit-test.h"
+#include "lib-oid.h"
```

Comment:
This header does not need to be included as it only contains the "old" assertions used by the other unit testing framework, right?